install/clax_install.sh

```diff
@@ -68,15 +68,16 @@ function generate_inetd {
     echo "clax stream tcp nowait $username $dir/clax clax -l $dir/clax.log" > $dir/inetd.clax
     echo "Generating xinetd.clax file"
     cat >$dir/xinetd.clax <<EOX
-service clax {
```

Comment:
Yes, it is. At least in the Raspbian provided xinetd. And I think it should not hurt other platforms.